### PR TITLE
Fix a bunch of bugs

### PR DESCRIPTION
**Pull request by yizheliu-amazon**
*Issue #, if available:*

*Description of changes:*
1. Fix issue that live chart flashes with multiple contents even data is still loading.
2. And correct the link to AD documentation
3. Fix breadcrumb on Dashboard

Fixed breadcrumb
Screen Shot 2020-04-29 at 5 01 23 PM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/pages/Dashboard/Container/DashboardOverview.tsx [hunk 1]
```diff
@@ -163,12 +176,10 @@ export function DashboardOverview() {
...
-    // this is needed because on Dashboard.tsx, getDetectorList API is called as 1s time, but it only gets 1 detector
-    // and such result is rendered to this component since this component also relies on result from getDetectorList API.
-    // And after this component does call getDetectorList to get all
-    // detectors, we need to refresh the page with result from 2nd call
-    // TODO: we need to implement namespacing for redux as per https://tiny.amazon.com/3eszqzpx/stacques4290
-    // Issue link: https://github.com/opendistro-for-elasticsearch/anomaly-detection-kibana-plugin/issues/23
+    chrome.breadcrumbs.set([]);
+  });
+
+  useEffect(() => {
```
Comment (ohltyler) [inline public/pages/Dashboard/Container/DashboardOverview.tsx:182]: why are you removing the breadcrumbs here? I thought we wanted to keep them for the dashboard and detector list page

File: public/pages/Dashboard/Container/DashboardOverview.tsx [hunk 2]
```diff
@@ -182,58 +193,75 @@ export function DashboardOverview() {
...
-        </EuiFlexItem>
-        <EuiFlexItem grow={3}>
-          <AnomalousDetectorsList selectedDetectors={currentDetectors} />
-        </EuiFlexItem>
-      </EuiFlexGroup>
+          <EuiSpacer />
+          <EuiFlexGroup justifyContent="spaceBetween">
+            <EuiFlexItem grow={6}>
+              <AnomaliesDistributionChart
+                allDetectorsSelected={true}
```
Comment (ohltyler) [inline public/pages/Dashboard/Container/DashboardOverview.tsx:258]: looks like `allDetectorsSelected` is never used when rendering the `AnomaliesDistributionChart` in `AnomaliesDistribution.tsx`. Maybe remove this prop field entirely?

File: public/pages/Dashboard/Container/DashboardOverview.tsx (hunk 1 above)
Comment (yizheliu-amazon) [reply public/pages/Dashboard/Container/DashboardOverview.tsx]: thanks for the catch. changed to show dashboard in breadcrumbs

File: public/pages/Dashboard/Container/DashboardOverview.tsx (hunk 2 above)
Comment (yizheliu-amazon) [reply public/pages/Dashboard/Container/DashboardOverview.tsx]: good catch. Removed.

File: public/pages/Dashboard/Components/EmptyDashboard/EmptyDashboard.tsx [hunk 3]
```diff
@@ -32,7 +32,7 @@ export class EmptyDashboard extends Component<{}, {}> {
             <p>
               Read about{' '}
               <EuiLink
-                href="https://github.com/opendistro-for-elasticsearch/anomaly-detection"
+                href="https://opendistro.github.io/for-elasticsearch-docs/docs/ad/"
```
Comment (ylwu-amzn) [inline public/pages/Dashboard/Components/EmptyDashboard/EmptyDashboard.tsx:35]: I remember we have another place using wrong link. Can you help fix it? You can search with `href="https://github.com`

Comment (yizheliu-amazon) [reply public/pages/Dashboard/Components/EmptyDashboard/EmptyDashboard.tsx:35]: sure. It seems to be there at `Features.tsx`. Will fix it

**Pull request merged**